apertium-tur-uzb.tur-uzb.dix

```diff
@@ -948,7 +949,6 @@
 <e><p><l>tohum<s n="n"/></l><r>urugʻ<s n="n"/></r></p></e>
 <e><p><l>yahudi<s n="n"/></l><r>yahudiy<s n="n"/></r></p></e>
 <e><p><l>dağ<s n="n"/></l><r>togʻ<s n="n"/></r></p></e>
-<e><p><l>kul<s n="n"/></l><r>qul<s n="n"/></r></p></e>
```

Comment:
I assume this is just some deduplication?

Reply:
Yep